README.md

```diff
@@ -292,8 +289,12 @@ Fabian Greif ([\@dergraaf](https://github.com/dergraaf)),
 Kevin Läufer ([\@ekiwi](https://github.com/ekiwi)),
 Martin Rosekeit ([\@thundernail](https://github.com/thundernail)),
 Daniel Krebs ([\@daniel-k](https://github.com/daniel-k)),
-Georgi Grinshpun ([\@georgi-g](https://github.com/georgi-g))
-and <!--authorcount-->33<!--/authorcount-->  more contributors.
+Georgi Grinshpun ([\@georgi-g](https://github.com/georgi-g)),
+Christopher Durand ([\@chris-durand](https://github.com/chris-durand))
+David Hebbeker ([\@dhebbeker](https://github.com/dhebbeker)),
+Thorsten Lajewski ([\@TheTh0r](https://github.com/TheTh0r)),
+Mike Wolfram ([\@mikewolfram](https://github.com/mikewolfram))
+and [many more contributors][contributors].
```

Comment:
Continued discussion from https://github.com/modm-io/modm/pull/592#discussion_r604253579:

I've made >20 the new cutoff and linked to the GitHub contributors page. It's arbitrary, but I also don't have a better way and also I don't really want to spend too much time on it. The releases/changelog explicitly call out the contributors again and also the individual contributions in the changelog.

cc @chris-durand